telegram/ext/_jobqueue.py

```diff
@@ -779,6 +779,19 @@ def __init__(
         self._job = cast("APSJob", None)  # skipcq: PTC-W0052
 
     def __getattr__(self, item: str) -> object:
+        """Overrides :py:meth:`object.__getattr__` to get specific attribute of
+        object :class:`telegram.ext.Job` object if exists.
```

Comment:
```suggestion
        the :class:`telegram.ext.Job` object or of its attribute :class:`apscheduler.job.Job`, if exists.
```

Sorry, my previous comment wasn't well formulated …